winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,23 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', '$stateParams',
+  function($scope, claimService, $stateParams) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+    $scope.allForms = [
```

Comment:
Why do we need to go back to hardcoded data here? It seems like a step in the wrong direction